.circleci/config.yml

```diff
@@ -32,6 +32,10 @@ jobs:
             "$CIRCLE_PROJECT_REPONAME" \
             "$CIRCLE_BUILD_URL" > version.json
 
+      - run:
+          name: Make sure megaphone image is fresh
+          command: docker-compose pull megaphone
```

Comment:
Actually, can we change this to pull all the dependencies outside "self". 
Even for things like `postgres:9.6` there are new versions we might want to get *within* the `9.6` version. E.g.

```yaml
- run:
    name: Pull dependencies to latest version (applicable for version pinned dependencies too)
    command: docker-compose pull megaphone db mysql memcached 
```